src/collectd-threshold.pod

```diff
@@ -163,6 +166,20 @@ generated for each value that is in the acceptable range. If set to B<false>
 (the default) then a notification is only generated if a value is in range but
 the previous value was not.
 
+=item B<PersistFrequency> I<Value>
+
+Sets how often notifications are generated if B<Persist> or B<PersistOK> is set
+to B<true>. This defaults to B<0>, which means every time a value comes in, a 
+notification is dispatched.
+If set to a positive integer, persistent notifications are suppressed that number
```

Comment:
Can *Value* be a time in seconds? The *interval* setting may be different for each metric, so this setting has very different meaning for them …